boards/arm/stm32l562e_dk/doc/index.rst

```diff
@@ -259,14 +259,30 @@ You should see the following message on the console:
 Building Secure/Non-Secure Zephyr applications with Arm |reg| TrustZone |reg|
 -----------------------------------------------------------------------------
 
-The TF-M integration sample :ref:`tfm_ipc` can be run on a STM32L562E-DK Discovery,
-using the ``stm32l562e_dk_ns`` target. When building a ``*_ns`` image with TF-M,
-a ``build/tfm/postbuild.sh`` bash script will be run automatically as a post-build step
-to make some required flash layout changes. The ``build/tfm/regression.sh`` script will
-need to be run to perform device initialization, and then run ``west flash --hex-file build/tfm_merged.hex``
-to flash the board.
-
-Check the ``build/tfm`` directory to ensure that the commands required by these scripts
+The TF-M integration sample :ref:`tfm_ipc` can be run on a STM32L562E-DK Discovery.
+In TFM configuration, Zephyr is run on the non-secure side. A non-secure image
+could be generated using ``stm32l562e_dk_ns`` as build target.
+
+.. code-block:: bash
+
+   $ west build -b stm32l562e_dk_ns
```

Comment:
Same here. The sample project path `samples/tfm_integration/tfm_ipc` is required?